src/src/ESPEasyCore/ESPEasyRules.cpp

```diff
@@ -1019,12 +1019,12 @@ bool findCompareCondition(const String& check, char& compare, int& posStart, int
 bool compareValues(char compare, float Value1, float Value2)
 {
   switch (compare) {
-    case '>' + '=': return Value1 >= Value2;
-    case '<' + '=': return Value1 <= Value2;
-    case '!' + '=': return Value1 != Value2;
-    case '>':       return Value1 > Value2;
-    case '<':       return Value1 < Value2;
-    case '=':       return Value1 == Value2;
+    case '>' + '=': return !definitelyLessThan(Value1, Value2);
+    case '<' + '=': return !definitelyGreaterThan(Value1, Value2);
+    case '!' + '=': return !essentiallyEqual(Value1, Value2);
```

Comment:
Am I missing something here?
Does not `case '!' + '=':` check for `^` (0x5e) this way, and for 'y' and '{' the two above.
*(although, this is a valid python syntax :)*

Reply:
Hmm have not checked if the ASCII value of those adds up to a character we also supported as an operator.

Reply:
Fixed in this pending PR: https://github.com/TD-er/ESPEasy/commit/16f00764b165c92abd0f13b70e0f655e7a29c610